terraform/ndelius-test-automation.tf

```diff
@@ -14,3 +14,20 @@ module "ndelius-test-automation" {
     },
   ]
 }
+
```

Comment:
This needs to be changed. Should be in a block as follows:

Reply:
{
      github_user  = "<github username>"
      permission   = "push"  #  pull|push|admin
      name         = ""  #  The name of the person behind github_user
      email        = ""  #  Their email address
      org          = ""  #  The organisation/entity they belong to
      reason       = ""  #  Why is this person being granted access?
      added_by     = ""  #  Who made the decision to grant them access? e.g. 'Awesome Team <awesome.team@digital.justice.gov.uk>'
      review_after = ""  #  Date after which this grant should be reviewed/revoked, e.g. 2021-11-26
    },